components/Common/Sidebar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import { useState, type FC } from 'react';
+
+import type { ActiveItem, SidebarGroupType } from '@/types/sidebar';
```

Comment:
Instead of creating a new type file for components, we prefer to add it to the existing component. You can examine existing components as examples under `/components/Common`